terragrunt/aws/generate_sbom/ecr.tf

```
@@ -15,4 +15,41 @@ resource "aws_ecr_repository" "generate_sbom" {
     Terraform             = true
     Product               = "${var.product_name}-${var.tool_name}"
   }
-}
\ No newline at end of file
+}
+
+resource "aws_ecrpublic_repository" "generate_sbom_public" {
+  provider        = aws.us-east-1
+  repository_name = "${var.product_name}/generate_sbom_public/trivy"
```

Thank you for taking care of this!  You can delete the old private ECR and take its name for this one:
```suggestion
  repository_name = "${var.product_name}/generate_sbom/trivy"
```